front/src/modules/rollingStock/components/RollingStockSelector/WithRollingStockSelector.tsx

```diff
@@ -1,9 +1,11 @@
 import React, { ComponentType, useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
-import { getRollingStockID, getRollingStockComfort } from 'reducers/osrdconf/selectors';
 import { RollingStockWithLiveries, osrdEditoastApi } from 'common/api/osrdEditoastApi';
 import { useTranslation } from 'react-i18next';
 import RollingStock2Img from 'modules/rollingStock/components/RollingStock2Img';
+import { useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Reply:
Done